### PR TITLE
[MIG] project_task_dependency: Migration to 11.0

### PR DESCRIPTION
**Pull request by PCatinean**

### CODE REVIEW

File: project_task_dependency/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+=====
+
+To use this module, you need to:
+
+#. Go to a task;
+#. click on 'Dependencies'.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/10.0
```
Comment (astirpe) [inline project_task_dependency/README.rst]: 11.0

File: project_task_dependency/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (astirpe) [inline project_task_dependency/README.rst]: Can you please adopt the png image, according to the latest [readme template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)?

**Review by astirpe — CHANGES_REQUESTED** [review]

File: project_task_dependency/README.rst (hunk 1 above)
Comment (PCatinean) [reply project_task_dependency/README.rst]: Done

File: project_task_dependency/README.rst (hunk 2 above)
Comment (PCatinean) [reply project_task_dependency/README.rst]: Done

**Review by astirpe — APPROVED** [review]
Code review only, not tested.

File: project_task_dependency/README.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Known issues / Roadmap
+======================
+
+* Constraint on stage progression based on the task dependencies
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/140/issues>`_. In case of trouble, please
```
Comment (RoelAdriaans) [inline project_task_dependency/README.rst]: Wrong url to issues, should be https://github.com/oca/project/issues

File: project_task_dependency/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# © 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Project Task Dependencies',
+    'version': '11.0.1.0.1',
```
Comment (RoelAdriaans) [inline project_task_dependency/__manifest__.py]: Version  11.0.1.0.0

**Review by RoelAdriaans — CHANGES_REQUESTED** [review]

**Comment by RoelAdriaans** [comment]
Functional tested :+1:

**Comment by PCatinean** [comment]
@RoelAdriaans thanks for the review, changes done.

**Review by RoelAdriaans — APPROVED** [review]

**Review by wtaferner — APPROVED** [review]
LGTM

File: project_task_dependency/views/project_task_view.xml [hunk 5]
```diff
@@ -13,10 +13,9 @@
                     <field name="recursive_dependency_task_ids" invisible="1"/>
                     <field name="dependency_task_ids"
                            domain="[
-                                   ('id', '!=', id),
-                                   ('id', 'not in', recursive_dependency_task_ids and recursive_dependency_task_ids[0][2] or [])
-                                   ]"
-                    >
+                                ('id', '!=', id),
```
Comment (pedrobaeza) [inline project_task_dependency/views/project_task_view.xml]: Better to put the domain in only one line for avoiding strange indentation on backend when editing the view

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please squash your 2 migration commits in one and I'll merge + little nitpicking

File: project_task_dependency/views/project_task_view.xml (hunk 5 above)
Comment (PCatinean) [reply project_task_dependency/views/project_task_view.xml]: Done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**